### PR TITLE
French translation 

### PR DESCRIPTION
**Pull request by merwok**

### CODE REVIEW

File: admin_interface/locale/fr/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+
+#: templates/admin/filter.html:13
+#: templates/admin_interface/dropdown_filter.html:4
+#, python-format
+msgid " By %(filter_title)s "
+msgstr " Par %(filter_title)s "
+
+#: templates/admin/popup_response.html:3
+msgid "Popup closing..."
+msgstr "Fermeture de la modale..."
```
Comment (merwok) [inline admin_interface/locale/fr/LC_MESSAGES/django.po]: I would like to review this string in context before this is merged!

Comment (fabiocaccamo) [reply admin_interface/locale/fr/LC_MESSAGES/django.po]: What do you mean?

Comment (merwok) [reply admin_interface/locale/fr/LC_MESSAGES/django.po]: I mean that I am not sure of my own translation :sweat_smile: 
I will run my app and see if what I wrote makes sense when I see it in the real context.

Comment (fabiocaccamo) [reply admin_interface/locale/fr/LC_MESSAGES/django.po]: I think that this can be merged, any translation can be updated in future...

Comment (merwok) [reply admin_interface/locale/fr/LC_MESSAGES/django.po]: SHIP IT

Comment (merwok) [reply admin_interface/locale/fr/LC_MESSAGES/django.po]: I think packaging still needs a little update to make sure the `.mo` files are shipped and installed.

Comment (fabiocaccamo) [reply admin_interface/locale/fr/LC_MESSAGES/django.po]: `.mo` files are included in package, as you can see in `MANIFEST.in`
A good idea could be compile messages at packaging time, to be sure that `.mo` files are always updated each time a translation is updated or a new localization is added.

Comment (merwok) [reply admin_interface/locale/fr/LC_MESSAGES/django.po]: I also see the `.mo` files in the wheel! :sparkles: :ferris_wheel:

**Pull request merged**